pages/common/phpcs.md

```diff
@@ -0,0 +1,35 @@
+# phpcs
+
+> Tokenize PHP, JavaScript and CSS files to detect violations of a defined set of coding standards.
+
+- Sniff the specified directory for issues:
+
+`phpcs {{path/to/directory}}`
+
+- Display a list of installed coding standards:
+
+`phpcs -i`
+
+- Specify a coding standard to validate against:
+
+`phpcs {{path/to/directory}} --standard {{standard}}`
+
+- Specify file extension(s) to include when sniffing:
```

Comment:
Is there a way to exclude files too?

Reply:
There's an `--ignore` option which can be used.

Reply:
Perhaps we could one of the lesser used options out for that, or include it in this example? It sounds useful to me. Thoughts?

Reply:
I guess maybe `--ignore` instead of `-l` (no recursion)? 🤔

Reply:
Yeah, I guess. I guess you can always use shell globs with it (i.e. `badfolder/**` will grab _everything_ in `badfolder/`)